automation/src/main/java/org/wso2/testgrid/automation/executor/JMeterResultCollector.java

```diff
@@ -60,11 +63,11 @@ public void sampleOccurred(SampleEvent sampleEvent) {
                                             "\", \"Status code\": \"",
                                             result.getResponseCode().replaceAll("\"", "\\\""),
                                             "\", \"Response Message\": \"",
-                                            result.getResponseMessage().replaceAll("\"", "\\\""),
-                                            "\", \"Sampler Data\": \"",
-                                            result.getSamplerData().replaceAll("\"", "\\\""), "\"}");
+                                            result.getResponseMessage().replaceAll("\"", "\\\""));
             // Persist result to the database
             testCaseUOW.persistTestCase(result.getSampleLabel(), testScenario, result.isSuccessful(), failureMessage);
+            logger.info(StringUtil.concatStrings(failureMessage, "\", \"Sampler Data\": \"",
+                    result.getSamplerData().replaceAll("\"", "\\\""), "\"}"));
```

Comment:
I think we need to print sampleLabel, testScenario along with this. Basically, we need more context.. :)

Reply:
Fixed in 1a609df